fgd/brush/trigger/trigger_reversespeed.fgd

```diff
@@ -1,6 +1,5 @@
-@SolidClass base(MomentumTrigger)
-	appliesto(MOMENTUM)
-= trigger_momentum_reversespeed : "Trigger for reversing the speed of the momentum player." 
+@SolidClass base(StaticTargetName, EnableDisable, Toggle)
+= trigger_reversespeed : "Trigger for reversing the speed of the momentum player." 
```

Comment:
Looks like the description is still momentum specific 😉

Reply:
Woops!